File: INICIANTE/JAVASCRIPT/1008.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
var prompt = function(texto) { return lines.shift(); };

var numerofun = prompt("NUMEROF");
var horast = prompt("HORAST");
var salarioph = prompt("SALARIOPH");
var salario = salarioph * horast;

console.log("NUMBER = " + numerofun);
console.log("SALARY = U$ " + salario.toFixed(2));

File: INICIANTE/JAVA/1008.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int A = sc.nextInt();
        int B = sc.nextInt();
        float C = sc.nextFloat();
        float salario = B * C;
        System.out.print("NUMBER = " + A +"\n");
        System.out.println(String.format("SALARY = U$ %.2f" , salario));
    }
} 
